ev3dev/sound.py

```diff
@@ -153,10 +161,37 @@ def beep_args(frequency=None, duration=None, delay=None):
         else:
             raise Exception("Unsupported number of parameters in Sound.tone()")
 
-    def play_tone(self, frequency, duration_ms, delay_ms=100, volume=100, play_type=PLAY_WAIT_FOR_COMPLETE):
+    def play_tone(self, frequency, duration, delay=0.1, volume=100,
+                  play_type=PLAY_WAIT_FOR_COMPLETE):
+        """ Play a tone, specified by its frequency, duration, volume and final delay.
+
+        Args:
+            frequency (int): the tone frequency, in Hertz
+            duration (float): tone duration, in seconds
+            delay (float): delay after tone, in seconds
+            volume (int): sound volume in percent (between 0 and 100)
+            play_type (int): one off Sound.PLAY_xxx play types (wait, no wait, loop)
+
+        Returns:
+            the sound playing subprocess PID when no wait play type is selected, None otherwise
+
+        Raises:
+            ValueError: if invalid value for parameter(s)
+        """
         self._validate_play_type(play_type)
+
+        if duration <= 0:
+            raise ValueError('invalid duration (%s)' % duration)
+        if delay <= 0:
```

Comment:
Why can't delay be `0`?

Reply:
It can be. Sorry for the copy/paste of this part of the sanity check code. I'll fix this ASAP.